### ISSUE TITLE
Starting VM immediately segfaults system?

### ISSUE THREAD

**Issue by biggestsonicfan** [QaidVoid/Complete-Single-GPU-Passthrough#27]
Greetings. Just trying this out and I've followed all directions to a T, but whenever the VM starts, I'm thrown to tty with the only visible errors being BTRFS errors with a final error of `Filesystem has been shut down due to the log error (0x2).` and `Please unmount the filesystem and rectify the problem(s).`. I can't actually see what errors were thrown because the errors cause logging to be disabled.

The VM previously booted fine. I'm on a SUSE host Win10 guest setup with Win10 as a separate physical partition. I can't attach `/var/log/libvirt/qemu/` as whatever is happening is occurring before logging.

See attached photo for visible error specifics:
![problem](https://user-images.githubusercontent.com/9744201/229383381-041b75ff-2c66-4fe4-be85-561034e903bd.jpeg)

EDIT: Trying to boot without a drive attached throws me to tty as well but at the login prompt. I think SUSE may handle sessions differently than described in the tutorial.

**Comment by QaidVoid** [comment]
I've no clue about this issue.

> The VM previously booted fine.

Was that without passthrough?

**Comment by biggestsonicfan** [comment]
> Was that without passthrough?

Passthrough gpu? Yes, it was without passthrough gpu. It was passthrough partition, however.

**Comment by QaidVoid** [comment]
Did some search on similar error. Couldn't find anything else other than the issue could be due to the lack of available storage space in the partition, which is probably not true in your case since it works fine without GPU passthrough..

I don't really have any idea about this issue..

**Comment by biggestsonicfan** [comment]
I'll play with some variables to try to isolate what's going on. I'm assuming it's something in the startup script, but that's a big assumption.

**Comment by biggestsonicfan** [comment]
Figured it out. Both `start.sh` and `stop.sh` Detach/Attach GPU devices from host. Since I literally copied and pasted, I was detaching `pci_0000_01_00_0` and `pci_0000_01_00_1` which appeared to be important to disk IO and was definitely not my graphics card. While I added the pci devices in the VM as PCI Host Devices, I did not change these to be the GPU and GPU Audio. I think I can move forward from here, but perhaps a little note in the scripts themselves or the Libvirt Hooks section to replace those values might be helpful for others going forward.
